docs/user_guide/further_reading.rst

```diff
@@ -0,0 +1,35 @@
+.. _further_reading:
+
+Further Reading
```

Comment:
I'm in favor of just creating a file "further_resources.rst" rather than just "further_reading.rst", because we can then include links to videos / toolkits etc. 

For the list of resources, I'd rather start with just a very simple list grouped into different kinds of resources, but I'd rather not provide the annotations (they're going to be harder to scale and they'll be necessarily subjective, so possibly a subject of discussion).

We can use high-level groupings:
* "Books" (the three below)
* "Academic papers" (including the ones below)
* "Popular writings" (including things like blog posts; maybe there's a better name than "Popular writings")
* "Video tutorials"
* "Online textbook", e.g., https://fairmlbook.org/
* "Academic conferences"

...

Reply:
Don't have entries for all those headings yet, but have gathered things under them and removed annotations.